### PR TITLE
Added nuspec file to Mvc.Analyzers assembly

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Part of aspnet/Universe#1083

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.nuspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <projectUrl>$projectUrl$</projectUrl>
+    <iconUrl>$iconUrl$</iconUrl>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    <tags>$tags$</tags>
+    <repository type="git" url="$repositoryUrl$" commit="$repositoryCommit" />
+    <dependencies>
+      <group targetFramework=".NETStandard1.3">
+        <dependency id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="2.6.1" exclude="Build,Analyzers" />
+        <dependency id="NETStandard.Library" version="1.6.1" exclude="Build,Analyzers" />
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.nuspec]: @natemcmaster confirming we don't need this reference. The project's going to get an implicit reference to a newer version anyway.

File: src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.nuspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <iconUrl>$iconUrl$</iconUrl>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    <tags>$tags$</tags>
+    <repository type="git" url="$repositoryUrl$" commit="$repositoryCommit" />
+    <dependencies>
+      <group targetFramework=".NETStandard1.3">
+        <dependency id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="$CodeAnalysisCSharpWorkspacesVersion$" exclude="Build,Analyzers" />
```
[the remark this answers is not in the document]
Comment (natemcmaster) [reply src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.nuspec]: You caught a bug. This dependency should have been PrivateAssets='All'. The references to roslyn are only used to produce the analyzer. They're not needed by the application's runtime in order to use the analyzer.

[the remark this answers is not in the document]
Comment (ajaybhargavb) [reply src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.nuspec]: Ah okay. I'll update it.

**Comment by ajaybhargavb** [comment]
Updated.

File: src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.csproj [hunk 3]
```diff
@@ -7,14 +7,37 @@
...
+        authors=$(Authors);
+        description=$(Description);
+        tags=$(PackageTags.Replace(';', ' '));
+        licenseUrl=$(PackageLicenseUrl);
+        projectUrl=$(PackageProjectUrl);
+        iconUrl=$(PackageIconUrl);
+        repositoryUrl=$(RepositoryUrl);
+        repositoryCommit=$(RepositoryCommit);
+        copyright=$(Copyright);
+        CodeAnalysisCSharpWorkspacesVersion=$(MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion);
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.Analyzers/Microsoft.AspNetCore.Mvc.Analyzers.csproj]: Can this be removed?

**Pull request merged**